config/kwikset/888.xml

```diff
@@ -0,0 +1,44 @@
+<Product Revision="1" xmlns="https://github.com/OpenZWave/open-zwave">
+  <MetaData>
+    <MetaDataItem name="OzwInfoPage">http://www.openzwave.com/device-database/0090:0541:0003</MetaDataItem>
+    <MetaDataItem name="ProductPic">images/kwikset/smartcode.png</MetaDataItem>
```

Comment:
Is the pic really the same as smartcode?

Reply:
no they are close but not exact, i put smartcode.png as a temp and forgot to go back and fix it.  i have updated the PR to include the image and updated the xml to use the correct image 200x200 png